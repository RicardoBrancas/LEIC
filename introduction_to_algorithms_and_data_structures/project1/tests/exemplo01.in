A LAX 10
A LIS 20
A OPO 30
F LAX LIS
G LAX OPO
G OPO LIS
G LIS LAX
I LIS 15
I LAX -8
X