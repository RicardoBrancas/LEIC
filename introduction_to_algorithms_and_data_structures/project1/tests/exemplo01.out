*Capacidade de LAX inalterada
5:3
